setup.cfg

```diff
@@ -1,2 +1,10 @@
 [bdist_wheel]
 universal=1
+
+[coverage:run]
+branch = True
+parallel = True
```

Comment:
Can you add a comment that the same options need to be set when building the command in test_gcovr.py?